File: go-backend/internal/model/vo/apikey.go
package vo

import "time"

type ApiKeyVO struct {
	ID           int64      `json:"id,string"`
	KeyValue     string     `json:"keyValue"`
	KeyName      string     `json:"keyName"`
	Status       string     `json:"status"`
	TotalTokens  int64      `json:"totalTokens,string"`
	LastUsedTime *time.Time `json:"lastUsedTime"`
	CreateTime   time.Time  `json:"createTime"`
}

File: python-backend/app/schemas/apikey.py
"""API key schemas."""

from __future__ import annotations

from datetime import datetime

from pydantic import Field

from app.schemas.common import CamelBaseModel, LongIdModel


class ApiKeyCreateRequest(CamelBaseModel):
    key_name: str | None = Field(default=None, alias="keyName")


class ApiKeyVO(LongIdModel):
    key_value: str | None = Field(default=None, alias="keyValue")
    key_name: str | None = Field(default=None, alias="keyName")
    status: str | None = None
    total_tokens: int | None = Field(default=None, alias="totalTokens")
    last_used_time: datetime | None = Field(default=None, alias="lastUsedTime")
    create_time: datetime | None = Field(default=None, alias="createTime")
